### PR TITLE
Updated Documentation on Custom Certificates and External Load Balancers

### PR DESCRIPTION
**Pull request by mburke5678**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1439231
Original PR: https://github.com/openshift/openshift-docs/pull/8779

### CODE REVIEW

**Comment by mburke5678** [comment]
@sdodson @ikke-t
You looked at another [custom cert PR](https://github.com/openshift/openshift-docs/pull/7607) for me; is this something you can review and comment on?

**Comment by sdodson** [comment]
My opinion is that we should not mention using an external load balancer to perform TLS termination. By mentioning this it opens us up to having to document how to configure CA and certificates for any number of load balancers. My preference is to stick with layer 4 load balancers where we don't have to deal with that problem.

**Comment by mburke5678** [comment]
@sdodson I commented-out the line pending any objections to remove it.

@jianlinliu PTAL

File: install_config/certificate_customization.adoc [hunk 1]
```diff
@@ -284,6 +284,65 @@ openshift_hosted_registry_routetermination=reencrypt <3>
...
+        keyFile: wildcard.example.com.key (2)
+        names:
+          - "openshift.example.com"
+  metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
+----
++
+<1> Path to the certificate and key files for the CLI and other API calls.
++
+<2> Path to the certificate and key files for the web console.
++
```
Comment (jianlinliu) [inline install_config/certificate_customization.adoc]: According to my knowledge, <1> is irrelevant to this topic, no need highlight it here, suggest to remove it.

Comment (mburke5678) [reply install_config/certificate_customization.adoc]: @jianlinliu  I removed the <1> and split the <2> Path to the certificate and key files... line.

<1> Path to the certificate files for the web console.
<2> Path to the key files for the web console.

**Comment by ikke-t** [comment]
Just responding as you pinged me in the thread.... I must say I don't have a real environment for myself to test these, nor I'd have time to build such. I raised the earlier issue as it was important customer case, and I was able to work on it. Unfortunately I don't have the experience to comment in this.

I know it's a mess to work with external LBs and different types of SSL terminations. We just went through a case where F5 did man in a middle type of termination, and would have needed guidance with that. But I agree, if we go there documenting the external LBs, it's endless swamp due the amount of alternatives.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Review by sdodson — APPROVED** [review]
/lgtm

File: install_config/certificate_customization.adoc [hunk 2]
```diff
@@ -284,6 +284,64 @@ openshift_hosted_registry_routetermination=reencrypt <3>
 +
 * Set to `passthrough` to terminate encryption at the destination. The destination is responsible for decrypting traffic.
 
+[[configuring-custom-certificates-lb]]
+== Configuring a Custom Certificate for a Load Balancer
+
+If your {product-title} cluster uses the default load balancer or an enterprise-level load balancer, 
+you can use custom certificates to make the web console and API available externally using a publicly-signed custom certificate, leaving the existing internal certificates for
```
Comment (kalexand-rh) [inline install_config/certificate_customization.adoc]: s/using/by using

That's really long. Maybe s/, leaving the existing internal certificates for the internal endpoints./. The internal endpoints still use the existing internal certificates.

File: install_config/certificate_customization.adoc [hunk 3]
```diff
@@ -284,6 +284,64 @@ openshift_hosted_registry_routetermination=reencrypt <3>
...
+  metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
+----
++
+<1> Path to the certificate file for the web console.
++
+<2> Path to the key file for the web console.
++
+[NOTE]
+====
+The `namedCertificates` section should be configured for the host name associated with the `masterPublicURL` and `oauthConfig.assetPublicURL` settings only.
```
Comment (kalexand-rh) [inline install_config/certificate_customization.adoc]: Configure the  `namedCertificates` section for only the host name that is associated with the `masterPublicURL` and `oauthConfig.assetPublicURL` settings.

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions, but this looks good!

File: install_config/certificate_customization.adoc (hunk 2 above)
Comment (mburke5678) [reply install_config/certificate_customization.adoc]: I am tempted to leave this as is; as that statement was exactly what the customer is asking for.

**Comment by mburke5678** [comment]
cherrypick enterprise-3.9

**Pull request merged**